### PR TITLE
Use common 0.2

### PR DESCRIPTION
**Pull request by khiltunen**
This PR modifies wrapper classes to obey  the changes made to simphony-common 0.2.0 version

### CODE REVIEW

**Comment by khiltunen** [comment]
@nathanfranklin do you have time to review these changes ?

**Comment by nathanfranklin** [comment]
@khiltunen , sure.  i will be able to start looking at it tomorrow

**Comment by nathanfranklin** [comment]
The requirements file ( https://github.com/simphony/simphony-numerrin/blob/use-common-0.2/requirements.txt#L1) needs to be updated: `0.1.3`->`0.2.0`

File: CHANGES.txt [hunk 1]
```diff
@@ -1,6 +1,11 @@
 Simphony-Numerrin CHANGELOG
 ======================
 
+Release 0.1.1
+-------------
+
+* Use latest version of of simphony-common (>=0.2.0) (#8)
```
Comment (nathanfranklin) [inline CHANGES.txt:7]: This version requirement could be added to setup.py at https://github.com/simphony/simphony-numerrin/blob/master/setup.py#L34

File: numerrin_wrapper/numerrin_mesh.py [hunk 2]
```diff
@@ -64,8 +65,7 @@ def __init__(self, name, mesh, pool):
         self._numFaceLabelToUuid = maps[3]
         self._numCellLabelToUuid = maps[4]
         # point data
-        for point in mesh.iter_points():
-            self.update_point(point)
+        self.update_points(list(mesh.iter_points()))
```
Comment (nathanfranklin) [inline numerrin_wrapper/numerrin_mesh.py]: `list` could be removed:
`self.update_points(mesh.iter_points())`

**Comment by nathanfranklin** [comment]
the importing of H5CUDS can be removed from `test_numerrin_wrapper.py` and `poiseuille.py`

**Comment by nathanfranklin** [comment]
@khiltunen ,  I took a look at the code and ran the tests and example.

Looks good.  I just left a few minor comments.  I like the changes that resolve both #3 and #5.  :+1:

**Comment by nathanfranklin** [comment]
Note that there are some pep8 violations.  Maybe we can run flake8 on travis before the testing  ( https://github.com/simphony/simphony-numerrin/blob/master/.travis.yml#L22-L23) to see them.

**Comment by khiltunen** [comment]
Thank's again for the review comments. Pushed those changes. Do you see that we can make PR now ?

**Comment by nathanfranklin** [comment]
:+1:

**Pull request merged**